### PR TITLE
OpenRTB 2.6 For Internal Model

### PR DESCRIPTION
**Pull request by SyntaxNode**
Upgrades the internal OpenRTB model to support 2.6. This will allow the core code and adapters to access OpenRTB 2.6 fields, but features (like schains, eids, gdpr, ccpa) will still be supported at 2.5 locations only until a future PR adds up convert and down convert support.

This PR changes adapter code. The changes are just simple type replacements except for the Rubicon adapter which relied upon EID.ID, an non-standard field. Removed with their approval and flagging them here (@bretg) to review the changes to their tests.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -16,7 +16,8 @@ require (
 	github.com/julienschmidt/httprouter v1.3.0
 	github.com/lib/pq v1.10.4
 	github.com/mitchellh/copystructure v1.2.0
-	github.com/mxmCherry/openrtb/v15 v15.0.1
+	github.com/mxmCherry/openrtb v13.0.0+incompatible
+	github.com/mxmCherry/openrtb/v16 v16.0.0-alpha.2
```
Comment (SyntaxNode) [inline go.mod:20]: Don't worry about the alpha status. The library is being kept in alpha until it's implemented Prebid Server to prove it works well.

File: openrtb_ext/request.go [hunk 2]
```diff
@@ -79,27 +81,8 @@ type ExtRequestCurrency struct {
 
 // ExtRequestPrebid defines the contract for bidrequest.ext.prebid.schains
 type ExtRequestPrebidSChain struct {
-	Bidders []string                     `json:"bidders,omitempty"`
-	SChain  ExtRequestPrebidSChainSChain `json:"schain"`
-}
-
-// ExtRequestPrebidSChainSChain defines the contract for bidrequest.ext.prebid.schains[i].schain
-type ExtRequestPrebidSChainSChain struct {
```
Comment (SyntaxNode) [inline openrtb_ext/request.go:87]: Goodbye `SChainSchain`!

**Review by bsardo — COMMENTED** [review]
LGTM, just need to merge to resolve the conflict.

**Review by bsardo — APPROVED** [review]

**Review by hhhjort — APPROVED** [review]
LGTM

**Pull request merged**